Source/WebGPU/WebGPU/CommandEncoder.mm

```diff
@@ -910,13 +910,15 @@ static bool refersToAllAspects(WGPUTextureFormat format, WGPUTextureAspect aspec
     auto aspectSpecificFormat = Texture::aspectSpecificFormat(destinationTexture.format(), destination.aspect);
     auto blockSize = Texture::texelBlockSize(aspectSpecificFormat);
```

Comment:
Did you consider declaring blockSize to be Checked<T>, or even changing Texture and related types to return Checked<T> for every integral type it currently returns?

If the type is Checked<T> from the start, then the compiler will track where it flows for you.

Reply:
that is a great idea, while `texelBlockSize` will never be greater than `16`, it is often used in computations involving `width`, etc so then we ensure those are checked